classes/locations.php

```diff
@@ -258,28 +258,28 @@ function geo_location( $post ) {
 
 					<!-- City / Town -->
 					<tr valign="top">
-						<td><label for="location_city"><?php _e( 'City/Town', 'simplemap' ); ?></label></td>
+						<td><label for="location_city"><?php _e( 'City/Town', 'SimpleMap' ); ?></label></td>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->